index.html

```diff
@@ -5,32 +5,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
-      rel="stylesheet"
-    />
     <title>Profile Rapheal Okonta</title>
   </head>
   <body>
     <header>
       <nav class="nav_bar">
-        <a href="#" class="name_logo">RAPHEAL OKONTA</a>
-        <div class="link_header">
-        <div class="nav_header_link">
-          <a class="header_menu" href="#">Portfolio</a>
-          <a class="header_menu" href="#">About</a>
-          <a class="header_menu" href="#">Contact</a>
-          <img
+        <a href="#" class="nav_branding">RAPHEAL OKONTA</a>
+        <ul class="nav_menu">
+          <li class="nav_item">
+            <a href="#" class="nav_link">Portfolio</a>
+              <hr class="hr-line_break3">
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">About </a>
+          <hr class="hr-line_break3">
+                
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">contact</a>
+            <hr class="hr-line_break3">
+          </li>
+               <img
```

Comment:
- [ ] as mentioned in the requirements, each of the nav links should take you to a specific part on the page
- [ ] you are required to follow the design as much as possible. your mobile menu is supposed to cover the whole screen
**figma**
<img width="258" alt="image" src="https://user-images.githubusercontent.com/37901469/196715332-4cbad8ff-5c4b-4fc1-9f60-4b9a7cdff1a1.png">

**your design**
<img width="372" alt="image" src="https://user-images.githubusercontent.com/37901469/196715114-3fe6ba9a-6f81-4821-94f4-88392c118e39.png">